sdk/src/main/java/io/radar/sdk/model/RadarConfig.kt

```diff
@@ -2,21 +2,24 @@ package io.radar.sdk.model
 
 import org.json.JSONObject
 
-/**
- * Remote configuration object
- */
 internal data class RadarConfig(
     val meta: RadarMeta,
-    val featureSettings: RadarFeatureSettings
+    val featureSettings: RadarFeatureSettings,
+    val googlePlayProjectNumber: Long?,
+    val nonce: String?
 ) {
 
     companion object {
         private const val META = "meta"
         private const val FEATURE_SETTINGS = "settings"
+        private const val GOOGLE_CLOUD_PROJECT_NUMBER = "googleCloudProjectNumber"
+        private const val NONCE = "nonce"
 
         fun fromJson(res: JSONObject?) = RadarConfig(
             RadarMeta.fromJson(res?.optJSONObject(META)),
-            RadarFeatureSettings.fromJson(res?.optJSONObject(FEATURE_SETTINGS))
+            RadarFeatureSettings.fromJson(res?.optJSONObject(FEATURE_SETTINGS)),
+            res?.optLong(GOOGLE_CLOUD_PROJECT_NUMBER),
```

Comment:
@jkao - is this the right place for this? For this to work, `nonce` and `googleCloudProjectNumber` have to be the top level, but `featureSettings` in further down in `meta`. So the server response for this approach looks like

```
 {
   meta: { code: 200, trackingOptions: undefined, featureSettings: undefined },
   nonce: 'xxx',
   googleCloudProjectNumber: 690986093081
 }
```

Reply:
@tderouin @jkao We can grab these from `meta` instead, let me know